src/components/forms/create-question-dialog.tsx

```diff
@@ -12,10 +14,33 @@ import { Input } from "@/components/ui/input";
 import { Button } from "../ui/button";
 import { Label } from "../ui/label";
 import { PlusIcon } from "lucide-react";
-
 import { Textarea } from "../ui/textarea";
+import createQuestion from "@/lib/api/createQuestion";
+
+export function CreateQuestionDialog(surveyId: string) {
```

Comment:
That's not the way to pass properties to react components. Please check that out again

Reply:
https://react.dev/learn/passing-props-to-a-component